src/View/Widgets/BreadcrumbsEntry.vala

```diff
@@ -170,10 +170,7 @@ namespace Files.View.Chrome {
         }
 
         private void cancel_completion_dir () {
```

Comment:
Since this is only called in `reset` can we move this one-liner there? Especially since this function is now a misnomer